resources/events/responses/events.yml

```diff
@@ -0,0 +1,83 @@
+description: Returns an `event` object to the specified server.
+
+headers:
+  ratelimit-limit:
+    $ref: "../../../shared/headers/ratelimit.yml#/ratelimit-limit"
+  ratelimit-remaining:
+    $ref: "../../../shared/headers/ratelimit.yml#/ratelimit-remaining"
+  ratelimit-reset:
+    $ref: "../../../shared/headers/ratelimit.yml#/ratelimit-reset"
```

Comment:
Does a webhook count as a POST req and therefore get ratelimit headers? o.o

Reply:
A webhook is a POST request, yeah, so I'd assumed it also has the ratelimit headers––but good catch, I'll confirm that.